tests/lib/nested.sh

```diff
@@ -50,9 +50,26 @@ prepare_ssh(){
 
 create_assertions_disk(){
     mkdir -p "$WORK_DIR"
-    dd if=/dev/null of="$WORK_DIR/assertions.disk" bs=1M seek=1
-    mkfs.ext4 -F "$WORK_DIR/assertions.disk"
-    debugfs -w -R "write $TESTSLIB/assertions/auto-import.assert auto-import.assert" "$WORK_DIR/assertions.disk"
```

Comment:
Hm, would be nice to know why this stopped working - but fine to unblock things for now.